bom/build.gradle

```diff
@@ -1,81 +1,57 @@
-publishing {
-    publications {
-        maven(MavenPublication) {
-            pom.withXml {
-                def xml = asNode()
-
-                def profiles = new Properties()
-                profiles.load(new StringReader(new File("$projectDir/profiles.properties").text))
-
-                boolean micronautSnapshot = rootProject.version.toString().endsWith("-SNAPSHOT")
-                
-                xml.children().last() + {
-                    def mkp = delegate
-
-                    mkp.properties {
-                        for(dep in dependencyVersions) {
-                            def v = dep.value.version
-                            if(!v.toString().endsWith("-SNAPSHOT") || micronautSnapshot) {
-                                mkp."${dep.key}.version"(v)
-                            }
-                        }
-                    }
```

Comment:
Are the versions still being set as properties that can be overridden in applications? Seems like that is no longer the case

Reply:
It is not longer the case no, but since we switched the profile templates over to use Gradle's built in BOM support this feature was lost anyway so not point retaining it